doc/UltiSnips.txt

```diff
@@ -491,7 +491,17 @@ situation. This is useful in conditional mappings.
 This function returns 1 if UltiSnips can jump backwards in the current
 situation. This is useful in conditional mappings.
 
-3.5 Warning about missing python support           *UltiSnips-python-warning*
+
+3.5 Store persistence directory                         *g:UltiSnipsStoreDir*
+-------------------------------
+
+Some interpolation data stores need persistence ('file' and 'common' sotres).
+The g:UltiSnipsStoreDir should to be set to an existing directory to provide
```

Comment:
please be precise here about type of variable (probably string?), also the path should be absolute.